src/Admin/AbstractAdminExtension.php

```diff
@@ -129,6 +130,10 @@ public function configureActionButtons(AdminInterface $admin, $list, $action, $o
     public function configureDefaultFilterValues(AdminInterface $admin, array &$filterValues)
     {
     }
+
+    public function configureFilterParameters(AdminInterface $admin, FilterBag $filterBag): void
```

Comment:
Could you use events for this? We would like to reduce the Admin class

cc @greg0ire @OskarStark

Reply:
Do you mean dispatching the "sonata.admin.event.configure.filter_parameters" event directly at `AbstractAdmin::getFilterParameters()`? To achieve this, the event dispatcher should be injected into `AbstractAdmin`.

Reply:
Should we wait for #5498 in order to go forward? Or maybe should I cherry pick the required definitions in order to implement this feature?